client/src/redux/auth.js

```diff
@@ -1,65 +1,85 @@
-import API from '../utils/API'
+import API from "../utils/API";
 // global state for authentification
 
 const initialState = {
-    email: "",
-    password: "",
-    errorMessage: "",
-    token: "",
-    authenticated: false,
-}
+	email: "",
+	password: "",
+	errorMessage: "",
+	token: "",
+	authenticated: false,
+};
+
+export const registerComplete = (data) => (dispatch) => {
+	dispatch({
+		type: "CLEAR_ERROR",
+	});
+	dispatch({
+		type: "LOGIN",
+		payload: data,
+	});
+};
```

Comment:
add `registerComplete` to redux 

TODO: move the api from register to this file instead
```
export const register = (data) => (dispatch) => {
    API.register(data).then((result) => {
	    dispatch({
                type: "CLEAR_ERROR"
            })
            dispatch({
                type: "LOGIN",
                payload: result.data
            })
    });
}
```